components/Shared/NetworkSwitcherGlyph/index.jsx

```diff
@@ -11,14 +11,15 @@ const NetworkSwitcherButton = styled.button.attrs(() => ({
   flexShrink: '0',
   alignItems: 'center',
   justifyContent: 'center',
-  height: 6,
-  paddingLeft: 2,
-  padidngRight: 2,
+  paddingLeft: 4,
+  paddingRight: 4,
+  paddingTop: 1,
+  paddingBottom: 1,
   fontSize: 2,
   fontWeight: 1,
-  fontFamily: 'RT-Alias-Medium',
+  fontFamily: 'RT-Alias-Grotesk',
   border: 0,
-  borderRadius: 0
+  borderRadius: 4
```

Comment:
@as-dr wondering about this border radius - there are no 4th indexed border or border radius properties in our theme - is this pixels? Can we add a theme for this instead for consistency?

Reply:
Very weird. I added it in in another branch previously. Maybe it was overwritten? Will fix.

Reply:
The theme file does have these values, but they are under `radii` --

`  radii: ['0', '1px', '4px', '8px', '16px', '32px', '100px'],`

Reply:
My bad, didn't know that